garak/generators/nim.py

```diff
@@ -38,6 +38,7 @@ class NVOpenAIChat(OpenAICompatible):
         "top_p": 0.7,
         "top_k": 0,  # top_k is hard set to zero as of 24.04.30
         "uri": "https://integrate.api.nvidia.com/v1/",
+        "suppressed_params": {"n", "frequency_penalty", "presence_penalty"},
```

Comment:
Do we have a ref for these we can throw into a comment so we can keep it up to date?

Reply:
This was learned through trial by 422 / 400 error at the moment